src/main/resources/static/plugins/jsonpath/index-browser-umd.js

```diff
@@ -0,0 +1,1069 @@
+(function (global, factory) {
```

Comment:
What does this library do? Do we need it? "UMD" sounds like the introduction of a new modularization system, which I personally would like to avoid at this stage of development. Anyway, we should try to include only minimized scripts.